docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+[role="xpack"]
+[[ml-supervised-workflow]]
+= Introduction to supervised learning
+
+Elastic supervised learning allows you to train a {ml} model based on training 
+examples that you provide. You can then use your model to make predictions on 
+new data. This page describes the end-to-end workflow to clarify how you can 
```

Comment:
Not a mandatory change, but I think it's helpful to note this is a summary, not a walk-through/tutorial:
```suggestion
new data. This page summarizes the end-to-end workflow for
```